tools/build_config/Jenkinsfile

```diff
@@ -190,6 +190,32 @@ pipeline {
       }
     }
 
+    stage('Configure') {
+      if (isUnix()) {
+        sh '''
+          export PATH=${PATH}:~/.local/bin &&
```

Comment:
is this PATH modification required for Herbert? This was added to support the Python/doc steps in  Horace and is not set in any other build step.